docs/index.rst

```diff
@@ -61,8 +75,8 @@ Following this you can, for example, perform an constrained proximal primal dual
     x_est, = primal_dual.FBPD(y, options, None, None, h, p, None)         # Recover an estimate of x.
 
 
-CONTRIBUTORS
-==============================================
+Contributors 
+--------------------
 `Luke Pratley <https://www.lukepratley.com>`_, `Matthjis Mars <https://www.linkedin.com/in/matthijs-mars/>`_, `Matthew Price <https://scholar.google.com/citations?user=w7_VDLQAAAAJ&hl=en&authuser=1>`_.
```

Comment:
My name is spelled wrong here (and also in the readme)

Reply:
Corrected, sorry about that!